Template/createEdgeDevice.json

```diff
@@ -45,13 +45,21 @@
         "spiDev": {
             "type": "int",
             "allowedValues": [
+                0,
```

Comment:
Shall we change the allowedValues and default value of spiDev also on `azuredeploy.json`?

Reply:
Yes, did it. Thanks for the catch